From fc8eb3f642d0f6e438d43eaa929fd912c014af01 Mon Sep 17 00:00:00 2001
From: Kartikey Parmar <kartikeyx.r.parmar@intel.com>
Date: Wed, 15 Apr 2015 16:30:48 +0800
Subject: [PATCH 259/389] stmmac: quark: Add support for Reliance Creek
 platform

Define the PHY IDs present on the Intel Quark-based Reliance Creek platform

The PHY descriptors are added for "RelianceCreek" and "RelianceCreekSPU"

Signed-off-by: Kartikey Parmar <kartikeyx.r.parmar@intel.com>
Change-Id: I85edfe4bdf540b6ce85439d54e03c876b04cb9f6
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/4654
Reviewed-by: Fung, Joshua <joshua.fung@intel.com>
Tested-by: Fung, Joshua <joshua.fung@intel.com>
---
 drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
index f9af91e..96daeb1 100644
--- a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
+++ b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
@@ -158,6 +158,26 @@ static struct stmmac_pci_dmi_data quark_pci_dmi_data[] = {
 		.func = 7,
 		.phy_addr = 1,
 	},
+	{
+		.name = "RelianceCreek",
+		.func = 6,
+		.phy_addr = 1,
+	},
+	{
+		.name = "RelianceCreek",
+		.func = 7,
+		.phy_addr = 1,
+	},
+	{
+		.name = "RelianceCreekSPU",
+		.func = 6,
+		.phy_addr = 1,
+	},
+	{
+		.name = "RelianceCreekSPU",
+		.func = 7,
+		.phy_addr = 1,
+	},
 	{}
 };
 
-- 
1.9.3

